usaspending_api/api_contracts/contracts/v2/disaster/award/loans.md

```diff
@@ -0,0 +1,153 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Awards Spending Disaster/Emergency Funding via Loans [/api/v2/disaster/award/loans/]
+
+This endpoint provides insights on the Agencies awarding loans from disaster/emergency funding per the requested filters.
+
+## POST
+
+Returns loan spending details of Agencies receiving supplemental funding budgetary resources
```

Comment:
`Agencies` --> `loans`?

Reply:
I'm not quite following it. You prefer this?

> Returns Agency spending details of loans receiving supplemental funding budgetary resources

Reply:
I made adjustments, see if they are good enough for now